plugins/woocommerce-docs/client/util/url.ts

```diff
@@ -0,0 +1,8 @@
+export const isURL = ( url: string ): boolean => {
+	try {
+		new URL( url );
+		return true;
```

Comment:
In case we want to ensure HTTP: 

```suggestion
		const o = new URL( url );
		return o.protocol === "http:" || o.protocol === "https:";
```

Reply:
Good idea! this is just to catch typos really and messing up the protocol is easy to do. Will fix.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/38874/commits/ea0cb808df441350d7632ddbb61f9cc28aba1204